src/app/shared/auth-style.scss

```diff
@@ -0,0 +1,90 @@
+section {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 100vh;
+  overflow: hidden;
+}
+
+.image-container {
+  flex-basis: 48%;
+  height: 100%;
+}
+
+.auth-img {
+  width: 100%;
+  height: 100%;
+}
+
+.form-container {
+  flex-basis: 48%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
```

Comment:
Kindly make use of @mixin in scss and define a mixin globally use it through out the application 
@mixin flex-option($display, $justify-content, $align-items,$direction) {
  display: $display;
  justify-content: $justify-content;
  align-items: $align-items;
  flex-direction: $direction;
}

This will reduce your lines of code.